pkg/image/image.go

```diff
@@ -159,6 +159,19 @@ func New(image v1.Image, tmpDirGen *file.TempDirGenerator, contentCacheDir strin
 	return imgObj
 }
 
+func Read(image *Image) (*Image, error) {
```

Comment:
why create a static method to expose a public method on image when you already have the image?

Reply:
Removed, I just missed this.